.github/workflows/bump.yml

```diff
@@ -2,7 +2,7 @@ name: Bump
 on:
   workflow_dispatch:
   schedule:
-    - cron: "0 8 * * 1" # "At 08:00 on Monday"
+    - cron: 0 8 * * * # At 08:00 every day
```

Comment:
Since it has been switched to automatic releases, I think we can increase the release frequency.

For example, if I want to use a new feature introduced today in Ruff (or another tool), it would cause `validate-pyproject` to fail. The only options would be to remove this pre-commit hook first or wait for a week.